sefaria/model/text.py

```diff
@@ -5675,22 +5681,39 @@ def get_simple_term_mapping_json(self, rebuild=False):
         return self._simple_term_mapping_json
 
     def get_term(self, term_name):
+        """
+        Returns the full term, if mapping not present, builds the full term mapping.
+        :param term_name: String
+        :returns: full Term (Mongo Record)
+        """
         if not self._full_term_mapping:
             self.build_term_mappings()
         return self._full_term_mapping.get(term_name) if term_name in self._full_term_mapping else Term().load({"name": term_name})
 
 
 
     def get_topic(self, slug):
+        """
+        Returns the corresponding topic for a given slug
+        :param slug: String
+        :returns: topic map for the given slug Dictionary
+        """
```

Comment:
This actually returns a dict with keys "en" and "he" (see _build_topic_mapping()). The name of this function is confusing...

Reply:
Got it, thanks for pointing that out!